lib/src/test/java/io/ably/lib/test/realtime/RealtimeConnectFailTest.java

```diff
@@ -165,8 +168,9 @@ public void connect_while_disconnected() {
 	/**
 	 * Verify that the connection enters the disconnected state, after a token
 	 * used for successful connection expires
+	 * DISABLED: See issue https://github.com/ably/ably-java/issues/183
 	 */
-	@Test
+	/*@Test*/
```

Comment:
TBH I would prefer not to mix too many issues into the same PR. Can we keep this to the changes required for the issue stated in the title (or change the title)?